src/extension.ts

```diff
@@ -70,6 +70,12 @@ export function activate(context: ExtensionContext) {
       languageClient.sendNotification("quarkus/quarkusPropertiesChanged", event);
     }));
 
+    context.subscriptions.push(commands.registerCommand('quarkus/clientSettingsUpdate', (setting: string, value: any) => {
+      if (setting === 'validationUnknownExcluded') {
```

Comment:
full property key (quarkus.tools.validation.unknown.excluded) should be received thus checked